src/components/navigation/header/config/headerLinks.ts

```diff
@@ -39,15 +39,17 @@ export type NavigationLinkConfigProps = Omit<
 
 export const headerLinks = ({
   trackEvent,
+  experiments = {},
 }: {
   trackEvent?: (event: CustomEvent) => void;
+  experiments?: Record<string, string>;
 }): NavigationLinkConfigProps[] => [
   privateAutoScoutSellLinkConfig({ trackEvent }),
   privateMotoScoutSellLinkConfig({ trackEvent }),
   professionalSellLinkConfig({ trackEvent }),
   estimateLinkConfig({ trackEvent }),
   autoScoutAssureLinkConfig({ trackEvent }),
   motoScoutAssureLinkConfig({ trackEvent }),
-  electromobilityLinkConfig({ trackEvent }),
+  ...(experiments?.leasing === 'on' ? [leasingLinkConfig({ trackEvent })] : []),
```

Comment:
isn't it 
`...(experiments?.leasing === 'on' ? [leasingLinkConfig({ trackEvent })] : electromobilityLinkConfig({ trackEvent })),` ? With this, you are removing ev hub always